### DISCUSSION TITLE
Suricata hyperscan

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#9892] [Unsupported Versions]
Hi there

I'm looking to tune suricata as much as I can, and I was wondering if the hyperscan pattern matcher is compiled into the version supplied with SO, and if so whether it was set as the default matcher.

Thanks

Ross

**Comment by InfosecGoon** [comment]
It does not appear to be.

From inside of the so-suricata container on my standalone box:

 ```
/opt/suricata/bin/suricata --build-info | grep Hyper
  Hyperscan support:                       no
```

**Reply by rosswakelin** [reply]
This sounds like a PR for an enhancement then, unless there is a good reason why it wasn't compiled in
